views/build.gradle

```diff
@@ -23,13 +23,31 @@ android {
 
 dependencies {
 	compile fileTree(dir: 'libs', include: ['*.jar'])
+
+	// Let's ensure that we are using only one support library version
+	compile "com.android.support:design:$supportlib_version"
+	compile "com.android.support:support-v4:$supportlib_version"
 	compile "com.android.support:appcompat-v7:$supportlib_version"
-	compile 'uk.co.chrisjenx:calligraphy:2.3.0'
-	compile 'com.github.matt-allen:Android-Universal-Image-Loader:2.1.2'
-	compile 'com.yqritc:android-scalablevideoview:1.0.4'
 	compile "com.android.support:recyclerview-v7:$supportlib_version"
+	compile "com.android.support:transition:$supportlib_version"
+	compile "com.android.support:support-annotations:$supportlib_version"
+	compile "com.android.support:support-v4:$supportlib_version"
+
+	compile 'uk.co.chrisjenx:calligraphy:2.3.0', {
+		exclude group: 'com.android.support'
+	}
+
+	compile 'com.github.matt-allen:Android-Universal-Image-Loader:2.1.2', {
+		exclude group: 'com.android.support'
+	}
+
+	compile 'com.yqritc:android-scalablevideoview:1.0.4', {
+		exclude group: 'com.android.support'
```

Comment:
ScalableVideoView does not seem to have any lib/support lib as dependency?

Reply:
It uses appcompat-v7, see:
https://github.com/yqritc/Android-ScalableVideoView/blob/master/library/build.gradle

Reply:
It is in the build file but not in the final library binary referenced. You can check it by ./gradlew views:dependencies command line script. It seems like it is stripped when he builds the library as he only uses few compile time annotations from the support library, eg @nullable @nonnull . Nevertheless, I don't see much issue to have the exclude group there.

Reply:
Yes, you're right - it's not in the binary. Good spot! I can remove it if necessary